rust-runtime/aws-smithy-runtime-api/src/client/orchestrator.rs

```diff
@@ -100,6 +101,23 @@ impl RequestTime {
     }
 }
 
+/// Informs the orchestrator on whether or not the request body needs to be loaded into memory before transmit.
+///
+/// This enum gets placed into the `ConfigBag` to change the orchestrator behavior.
+/// Immediately after serialization (before the `read_after_serialization` interceptor hook),
+/// if it was set to `Requested` in the config bag, it will be replaced back into the config bag as
+/// `Loaded` with the request body contents for use in later interceptors.
```

Comment:
can you add docs about how this interacts with retries?

Reply:
Added in 7ed3d3fdb54c2d0b690b9e2f1d4ec2ab9dab5bef